docs/features/building/manage-blocks.md

```diff
@@ -4,24 +4,93 @@ title: Manage KintoBlocks
 
-To edit a KintoBlock follow the steps mentioned below:
+>Note: **Environment Variables** are not specified during the creation of KintoBlock.
 
-1. Open a KintoBlock.
+To edit a **Static Website** KintoBlock when `Build Required` toggle is `On` follow the steps mentioned below:
+
+1. Open a Static Website KintoBlock.
+
+      ![Settings Button](/docs/assets/kintoblock-settings.png)
+
+2. Click on **Settings** button displayed in sidebar to modify the information.
+
+3. Edit Basic Info of the KintoBlock like `KintoBlock Name`, `Language`, `Version`, and `Description`.
+
+      ![Editing Information of KintoBlock](/docs/assets/kintoblock-basic-info.png)
+
+4. Click on the **Save Changes** button.
+
+To edit a **Static Website** KintoBlock when `Build Required` toggle is `Off` follow the steps mentioned below:
+
+1. Open a Static Website KintoBlock.
+
+2. Click on **Settings** button displayed in the sidebar to modify the information.
+
+3. Edit Basic Info of the KintoBlock like `KintoBlock Name` and `Description`.
+
+   ![Editing Information of KintoBlock](/docs/assets/kintoblock-edit-info.png)
+   
+4. Click on the **Save Changes** button.
+
+To edit a **Dynamic Website** KintoBlock, follow the steps mentioned below:
+
+1. Open a Dynamic Website KintoBlock.
+
+2. Click on **Settings** button displayed in the sidebar to modify the information.
+
+3. Edit Basic Info of the KintoBlock like `KintoBlock Name`, `Language`, `Version`, and `Description`.
 
-2. Click on the **pencil icon** displayed beside the title to modify the Description and Display Name of the KintoBlock.
+4. Click on the **Save Changes** button.
 
-    ![Screenshot](/docs/assets/display-name.png)
+To edit hardware configuration for website, follow the steps mentioned below:
 
-3. Scroll down to **Builds & Repository** section.
+1. Open a Website KintoBlock.
 
-4. Expand **Advanced Options** to view the `Port`, `Build command`, `Run`, and `Doc` fields
+2. Click on **Settings** button displayed in the sidebar to modify the Hardware Configuration.
+
+3. Edit the default memory limit.
+
+      ![Hardware Configuration](/docs/assets/kb-hardware-configuration.png)
+
+4. Click on the **Save Changes** button.
+
+[Click Here](/docs/features/deploying/environment-variables) to Know more about **Environment Variables**.
+
+>Note: Above mentioned steps can be followed for **Microservice** and **Custom Service** for editing the properties like `KintoBlock Name`, `Language`, `Version`, `Description`, `Environment Variables` and `Hardware Configuration`.
+
+## Advanced Options for KintoBlock
+
+Advanced Options is used modify the KintoBlock with properties like `Protocol`, `Port`, `Build command`, `Start command` and `Doc` format are auto-filled based on the chosen language. These values are editable and can be changed if required.
```

Comment:
"used modify" -> "used to modify"

Remove Doc, we disabled it

Reply:
Done